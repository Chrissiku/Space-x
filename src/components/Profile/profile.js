/* eslint-disable linebreak-style */
/* eslint-disable no-unused-vars */
/* eslint-disable quotes */

const Profile = () => (
  <>
    <h1>Profile page</h1>
  </>
);

export default Profile;
