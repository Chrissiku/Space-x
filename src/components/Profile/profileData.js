/* eslint-disable linebreak-style */
/* eslint-disable comma-dangle */
/* eslint-disable no-unused-vars */
/* eslint-disable quotes */

import { useSelector } from "react-redux/es/exports";

const ProfilesData = () => {
  const profileParts = useSelector((state) => [
    {
      title: "My Rockets",
      data: state.rockets.filter((rocket) => rocket.reserved),
    },
  ]);
};

export default ProfilesData;
