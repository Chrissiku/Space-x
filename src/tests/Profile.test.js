/* eslint-disable linebreak-style */
/* eslint-disable comma-dangle */
/* eslint-disable quotes */
import React from "react";
import TestRenderer from "react-test-renderer";
import { Provider } from "react-redux";
import store from "../Redux/ConfigureStore";
import Profile from "../components/Profile/profile";

it("Profile render correctly", () => {
  const Tree = TestRenderer.create(
    <Provider store={store}>
      <Profile />
    </Provider>
  );
  expect(Tree).toMatchSnapshot();
});
