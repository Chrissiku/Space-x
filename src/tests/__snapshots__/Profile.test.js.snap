// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile render correctly 1`] = `
<div
  className="profile-container container"
>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <h4>
        My Missions
      </h4>
      <div
        className="list-group"
        onKeyDown={[Function]}
      />
    </div>
    <div
      className="col"
    >
      <h4>
        My Rockets
      </h4>
      <div
        className="list-group"
        onKeyDown={[Function]}
      />
    </div>
  </div>
</div>
`;
